proxy/proxy.go

```diff
@@ -1631,7 +1632,15 @@ func (p *Proxy) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	p.tracing.setTag(span, HTTPRemoteIPTag, stripPort(r.RemoteAddr))
 	p.setCommonSpanInfo(r.URL, r, span)
-	r = r.WithContext(ot.ContextWithSpan(r.Context(), span))
+
+	ctxWithSpan := ot.ContextWithSpan(r.Context(), span)
+
+	// https://pkg.go.dev/go.opentelemetry.io/otel/bridge/opentracing#readme-interop-from-trace-context-from-opentracing-to-opentelemetry
+	if bridgeTracer, ok := p.tracing.tracer.(*otBridge.BridgeTracer); ok {
+		ctxWithSpan = bridgeTracer.ContextWithSpanHook(ctxWithSpan, span)
+	}
```

Comment:
hm.. that's interesting.
I don't understand how it works now, when we provide bridgeTracer through skipper options.